src/CLICommand/parameters_helper.ts

```diff
@@ -224,11 +224,16 @@ export class ParametersHelper {
 	 * @returns {string | undefined}
 	 * Returns the string value for the specific parameter; throws an error if not set
 	 */
-	public getRequiredParameterValue(parameterName: ParameterName): string {
+	public getRequiredParameterValue<T = string>(
+		parameterName: ParameterName,
+		// eslint-disable-next-line @typescript-eslint/no-explicit-any
+		closure: (input: any) => T = (input: any) => input as T
+	): T {
+		// FIXME: it could also return an array or a boolean!
```

Comment:
We will surely implement different methods for reading boolean & array parameters. So this function may throw when reading a boolean/array